### PR TITLE
comecando estrutura dados IDs

### PR DESCRIPTION
**Pull request by pietro1704**

### CODE REVIEW

File: challengeFinal/Game.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//
+//  Game.swift
+//  challengeFinal
+//
+//  Created by Pietro Pugliesi on 14/10/20.
+//
+
+import Foundation
+
+class Game{
```
Comment (guilhermeShima) [inline challengeFinal/Game.swift]: Fix space between class name and "{"

File: challengeFinalComponents/gameComponents/StoryNodeManager.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//
+//  StoryNodeManager.swift
+//  challengeFinal
+//
+//  Created by Pietro Pugliesi on 14/10/20.
+//
+
+import Foundation
+
+struct StoryNodeManager{
```
Comment (guilhermeShima) [inline challengeFinalComponents/gameComponents/StoryNodeManager.swift]: Fix space between "{"

File: challengeFinalTests/StoryNodesTest.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+//
+//  Created by Pietro Pugliesi on 14/10/20.
+//
+
+import XCTest
+@testable import challengeFinal
+
+class StoryNodesTest: XCTestCase {
+
+
```
Comment (guilhermeShima) [inline challengeFinalTests/StoryNodesTest.swift]: Remove white spaces.

File: challengeFinalComponents/gameComponents/StoryNodeManager.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    var storyNodes = [StoryNode]()
+    
+    func getNodeByID(id:NodeID) -> StoryNode?{
+        let nodeIndex = storyNodes.firstIndex { (node) -> Bool in
+            node.id == id
+        }
+        guard nodeIndex != nil else { return nil}
```
Comment (guilhermeShima) [inline challengeFinalComponents/gameComponents/StoryNodeManager.swift]: Fix space between "{"

**Review by guilhermeShima — COMMENTED** [review]
Nice job!

File: challengeFinalTests/StoryNodesTest.swift [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+//
+
+import XCTest
+@testable import challengeFinal
+
+class StoryNodesTest: XCTestCase {
+
+    
+
+    override func setUpWithError() throws {
```
Comment (guilhermeluisdomingues) [inline challengeFinalTests/StoryNodesTest.swift]: Remove unused functions

File: challengeFinalTests/StoryNodesTest.swift [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        // This is an example of a functional test case.
+        print(node1)
+        print(node2)
+        print(node3)
+        print(node4)
+        print(node5)
+
+    }
+
+    func testPerformanceExample() throws {
```
Comment (guilhermeluisdomingues) [inline challengeFinalTests/StoryNodesTest.swift]: Remove unused functions

File: challengeFinalTests/StoryNodesTest.swift [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+    
+        
+    }
+
+    override func tearDownWithError() throws {
```
Comment (guilhermeluisdomingues) [inline challengeFinalTests/StoryNodesTest.swift]: Remove unused functions

**Review by guilhermeluisdomingues — APPROVED** [review]
Good!

**Pull request merged**